Solutions/STM32F4DISCOVERY/scatterfile_gcc_missing_symbols.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <ScatterFile xmlns="http://schemas.microsoft.com/netmf/ScatterfileSchema.xsd">
+  <!-- 
+    - using newlib-nano adds libnosys.a, which calls sbrk, which requires the 'end' symbol
+    - this is pulled in by a _printf imported in function _Crypto_StepRSAOperation @ Crypto.lib(rsa_zenith.obj) 
+  -->
+  <Provide Name="end" />

```

Comment:
That's rather great news (that you've already got mbed TLS working with NETMF) - can you merge it? :)

The question is, how many devices actually use the proprietary crypto hash function?

I guess leaving crypto libs for another minor version release is not much of an issue, they can be always removed later.